packages/commercetools/theme/pages/Category.vue

```diff
@@ -491,6 +491,7 @@ export default {
   &__list {
     display: flex;
     flex-wrap: wrap;
+    margin-top: 1.875rem - 0.5rem;
```

Comment:
I actually changed that line in a previous PR
```suggestion
    margin-top: 0;
```